sdk/src/types/block/output/feature/mod.rs

```diff
@@ -188,6 +208,17 @@ impl Features {
         Ok(Self(features))
     }
 
+    /// Creates a new [`Features`] from an ordered set.
+    pub fn from_set(features: BTreeSet<Feature>) -> Result<Self, Error> {
```

Comment:
better to impl it as IntoIter<> i thnk

Reply:
But then you lose the ordered/unique properties

Reply:
then we wouldnt need this method if we just implement it in the `Impl TryFrom<BTreeSet<Feature>> for Features {`

Reply:
Either way we have to define how we handle already ordered/unique data vs questionable data.